### ISSUE TITLE
And here it is, official update rolled out and mods are now officialy broken with MO (EDIT: found a way to play with MO just fine)

### ISSUE THREAD

**Issue by VijuaruKei** [TanninOne/modorganizer#663]
Thanks Bethesda. All of that for that crappy Bethesda.net with that terrible UI.

i haven't found any workaround to get MO working, and i really don't wan't to use NMM (and having to install all my mod again..)

Tried to put * before every esp file in the MO plugin.txt = Mods are still disabled

Tried to put * before every esp file both in the MO plugin.txt AND the Fallout4 one (the one at APPDATA LOCAL blablabla) = Game crash when i load a save.

Tried to activate mod from the in game new mod menu = Game crash when i load a save.

Thanks Again Bethesda. well no more Fallout 4 for me until TanninOne release an update :/ (not that i'm asking you to go quick he, take your time)

**Comment by LawLeewer** [comment]
Maybe your game is crashing because of an incompatible mod and not because of Mod Organizer.
I was pretty pissed at everything because I was having the same problem, then I deactivated Def_UI and everything works fine.
It IS pretty inconvenient to use MO with the new system that bethesda implemented because you have to activate the mods in the in-game menu everytime you want to play, but it works.

**Comment by VijuaruKei** [comment]
Well i do not think that i use any HUD mod, i don't use Def_UI too

here is my ESP list from the plugin.txt just in case

# This file was automatically generated by Mod Organizer.
Fallout4.esm
Homemaker.esm
ArmorKeywords.esm
DLCRobot.esm
DLCworkshop01.esm
Simple Intersection.esp
DeadBodyCollision.esp
extendedLightsWS.esp
chem redux.esp
MojaveImports.esp
MoBettaScrap.esp
CBBE.esp
dD-Enhanced Blood.esp
BetterSettlers.esp
BetterSettlersNoLollygagging.esp
LongerPowerLinesInf.esp
TrueStormsFO4.esp
TrueStormsFO4-ClimateSettings.esp
TrueStormsFO4-Performance-Fog.esp
ScrapDeadThings.esp
OWR.esp
MojaveMusicRadio.esp
Cuts and Blunts 3.0.esp
Radiant Clouds and Fogs.esp
Lots More Female Hairstyles.esp
Reverb and Ambiance Overhaul.esp
AutomatronCountyCrossingFix.esp
Better Vendor Stalls.esp
keyNuker_keys_fr.esp
keyNuker_cards_fr.esp
OctaviusSentibar_Labels.esp
RadiantBirdsv05.esp
Modern Firearms.esp
Armorsmith Extended.esp
OWR_LouderStations12.esp
Reverb and Ambiance Overhaul - Automatron.esp
Simple Bug Fixes.esp
TrueReporter.esp

**Comment by VijuaruKei** [comment]
Well no matter what i disabled it never works >>

**Comment by darkanvas** [comment]
Just writing here to confirm that yesterday I was using MO2 fine on FO4, got home from work today and none of my mods are working. Seems to be the legitimate case that Bethesda have broken MO2 with some patch or other...

**Comment by OneSubtractOne** [comment]
Def HUD has a patch out that fixes the CTD caused by them. Get it on the nexus. From what I can see, MO regenerates it's plugins file on loading, so even if you manually put the asterisks in, it removes them for you.
I think the asterisks are Bethesda's way of checking if a mod is activated? Going in to the mods option and then going to your library of mods gives you a full list of the mods but they're all not activated. I just hit enter on each one and then had no problems loading my game mods included. But you have to do that every single time. Which isn't much fun.

**Comment by VijuaruKei** [comment]
Well i found a way to make it works

Some mod make the game crash, it's NOT mod organizer fault, its the MOD fault

some mod that make the game crash : 
Key Nuker
Def HUD

but Def Hud has been updated lke OneSubtractOne just said.

So if the game crash after the loading it means that you have a mod that not work with the last update.


And for making ModOrganizer works with the last update : 

Use this mod https://www.nexusmods.com/fallout4/mods/11479/?tab=2&navtag=https%3A%2F%2Fwww.nexusmods.com%2Ffallout4%2Fajax%2Fmodfiles%2F%3Fid%3D11479&pUp=1
extract file in the folder where plugin.txt is located

i'm talking about the Modorganizer one, not the fallout 4 one  (the appdata/local).
You need to go to your modorganizer install folder, then to the profil folder, and then, on the Fallout4 Folder.

Once it's done, you need to launch ModOrganizer, make sure to check every mod you wan't, and then open the  .bat from the profiles\Fallout folders. it will put the asterisks automaticly, once it's done close it.

Now you can run the game with ModOrganizer and play just fine, if the game crash during the loading it means again, that one of your mod don't work with the last fallout 4 update


the only issues is that i think doing this mess up with the load order of the ESP, because when i close the game everything in modorganizer is mixed up, so be sure to save the load order by clicking to the little harddrive icon at the upper right of the .esp tab on mod organizer so you can put everything back in order easly but clicking to the yellow arrow icon. 


Sorry in advance for my bad english, i'm french so it's not my native language.

**Comment by darkanvas** [comment]
I'm actually not getting any CTD myself, the mods have just stopped working. I haven't tried using any asterisks though. When I try to continue my game it says that I am continuing without content upon which my save relies (even though all the mods are activated in MO2).

I had one idea which was to copy my plugins.txt entries to my dlc.txt, save the file and then run F4SE from MO2 instead of Fallout4.exe directly, no dice. I did that because I read up that it used to be a fix when NMM stopped working previously, so I thought it was worth a try.

I am **assuming** any CTDs are caused because Bethesda's garbage in-game mod 'manager' (*cough* destroyer) is looking for the ESP/ESM/mod files in FO4s data folder - instead of where they're actually located re: MO2.

It is however, wild speculation on my part...

**Comment by michaelrw10** [comment]
im not getting any CTD, im just not able to get the plugins to load (no *.ESP files)
Same thing with me though, my MO (MO2, A4) has been working fine for weeks now. Then today, all of a sudden its not. It has to be the update, as it is the only thing that has changed since last night. I'll try some of these suggestions.

**Comment by michaelrw10** [comment]
UPDATE: Both methods work to get the plugins enabled. 

Method 1 (OneSubtractOne): Re-enabling all of the mods in the "Mods" interface in-game does work. After doing so, once you exit the game and go back to MO, all of my mods are disabled in the load order, but they are still in the proper order.

Method 2 (VijuaruKei): Using the BAT script to add the * asterisks before each entry in the plugins.txt that resides in the Profiles folder of your MO installation. This works too. However, you have to edit the BAT file to remove the CD command that instructs the script to look at your appdata folder. Right-click the fix_plugins.bat file, select 'Edit'. Once open, remove the line that reads 
" cd %localappdata%\fallout4 "
and then save it. Removing this line will make it so the script will only look for the plugins.txt file in the current folder. So make sure to place fix_plugins.bat in the same folder as the plugins.txt file that you want to alter. Once the plugins.txt has been successfully modified, you can launch FO4 and immediately load your game without having to use the "Mods" interface. As with Method 1, when you exit the game and return to MO, all of the mods will be disabled, but still in the correct order.

Now, as other have reported, I am now getting CTD when I load my savegame. This issue is completely separate from the plugins not loading. It is clear that a mod which used to be compatible is no longer compatible. As other have suggested, I will look at DEF_UI first, and report back what I find.

**Comment by AikoTM** [comment]
The .bat method is working just fine. :)

Now we just need something to remove this "modded" savegame thing.
I want achievements :(

**Comment by darkanvas** [comment]
Confirming that the bat method does not work for me.

Steps:

leave appdata version of plugins.txt and profile.txt as default
downloaded the fixer, put it in the MO2 profiles default folder (the one I am using)
opened MO2
restored proper load order and mod order with load buttons
edited the bat file to remove the line as specified
saved the bat file
ran the bat file with MO2 still open
Launched the game
CTD during opening video before main menu

**Comment by AikoTM** [comment]
> *Armorsmith Automatron.esp
> *Armorsmith Extended.esp

Shouldn't this be the other way around?

**Comment by darkanvas** [comment]
I will try that when I get home. However if that hasn't been causing problems pre patch I can't see why it would now! From what I understand mods break entirely or remain fine, it's not a case of changing load order to make the game work again. 

I think it's much more likely that mods I use which utilise dll files (like fo4 hotkeys) have now been broken or something like that.

Also my CTD isn't when I load my game, it's before the main menu even loads so something is critically messed up. 

I shouldn't have to go to TPB and download an old game version to fix bethesdas crap but I guess that's what's going to happen lol

**Comment by t41nt3d** [comment]
The batch is a guaranteed fix. I've been using it throughout the Beta, it means one of the mods you have isn't compatible or you have a conflict somewhere.

It's not Bethesda's fault in the slightest, this is just what happens when you mod a game that still gets patches, stuff gets broken.

**Comment by darkanvas** [comment]
Didn't happen anywhere near as much with Skyrim which was on the same engine :( guess we'll have to agree to disagree. I had the game just how I liked it then it all broke because of the patch. Obviously no one would expect me to be a raging Bethesda fan at the moment lol

Personally I'm just going to downgrade for an easier life until it all blows over. Thanks for trying to help though!

**Comment by AikoTM** [comment]
This is obviously a result from Bethesda trying to get Mods to consoles - there is no other reason for the new modding system.
Same with the fact that you can't get achievements with modded games now. This has been proven to be the result of an agreement between Microsoft and Bethesda in order to get modding allowed on XBONE.

I'm not against modding on consoles - eventho it will never be the same like modding on PC. Let those console guys have mods. But i hate that we on PC have to suffer for it now..

Sorry for being kinda Off-Topic.

**Comment by darkanvas** [comment]
I think you make some good points friend :)

For anyone else who is unwilling to accept uninstalling mods they link because of Bethesda's patch, for anyone who is looking to downgrade, the version list of Fallout 4 is here:

http://fallout.wikia.com/wiki/Fallout_4_patches

This is the patch that broke the game:

1.5.157 Beta, released on April 26, 2016

This is the version you will need to download to fix your game if you don't want to be messing around with bat files and all that other bullcrap:

1.5.151 Beta, released on April 20, 2016

**Comment by AikoTM** [comment]
I love the PC Modding Community.
Here is a fix for the Achievements - requires F4SE. 
https://www.nexusmods.com/fallout4/mods/12465/?

**Comment by valik2104** [comment]
It happened to Skyrim when Workshop went live, but MO wasn't still in alpha and updates came quick. Now that Beth.net is live, and the Ck is out, maybe we'll see a beta on Nexus.  I rather not play then use the nexus mod manager again.

**Comment by TheExenth** [comment]
Guys you can just create a Bethesda Account on their Website, login with it ingame to get to the new Modmenu and when you go to My Library you can enable all your mods which where installed with Mod Organizer

**Comment by michaelrw10** [comment]
@darkanvas -- your crash has nothing to do with the batch file. The batch file is only intended to load/enable your plugins at game launch through MO. If your game is crashing, it just means that you have a mod that is now incompatible with FO4 (like DEF_UI), or like others suggested, you have a load order problem. Either way, the CTD is unrelated to batch file

**Comment by michaelrw10** [comment]
I wonder if tannin had a donation fund and had all of the MO users make small donation, if that would help speed up his development. $1 from each user and he'd have thousands very quickly. Hell if probably donate $20 myself. I know this isn't his full time job, but still. I am dying for a stable MO2. I cannot imagine Bethesda gaming without MO. We've been spoiled, and I never want to go back to just Wrye, or even worse, NMM. Yuck

**Comment by kranky1260** [comment]
Hi I managed to get all my mods running with FO4 1.5.
1. sort your load order, than lock load order.
2. Leave MO opened and run FO4 via F4SE 
3. From main menu scroll to MODS activate your mods than escape to main menu and run your game

Once you are back in desktop go to MO and re-enable your mods and follow above steps to play your game again.
My CTD was cause by UXO, i did a trial and error with all mods until this (UXO) crash, since than no probs. Hope this helps someone.

**Comment by GSDFan** [comment]
Some good information here. I hope Tannin can use it. 

 I wanted to ask in if anyone has seen non-activated ESPs in the mods, My Library? I noticed this ESP, No more monochrome - Color setting for the Pipboy!.esp will not activate, but if I change it to No more monochrome - Color setting for the Pipboy.esp it will be activated.

It looks like the special character is to blame and not a length issue. I changed it to a longer name and it activates.

**Comment by michaelrw10** [comment]
hey gsdfan.. yeah it definitely sounds like its an issue with special characters. I may make a new post in the mo2 thread on step.. that thread hasnt had any new conversation since the latest update. Have you tested the special character theory with a different plugin?

**Comment by GSDFan** [comment]
I believe I test all allowable characters in a file name and all worked with the exception of the percent sign. If the percent sign was the first character in the name the game would CDT on load. if it was moved to another location the game would load the mod without a CDT.

**Comment by Astor1977** [comment]
Using either the bat option or the in game mod list (separately or together), the mods won't load properly.  I know the game is trying to load them because my toon's hand turns purple in the pipboy menu, and disappears entirely in 3rd person view.  It's really very frustrating.

**Comment by kranky1260** [comment]
Hi Astor1977
Just follow my steps above but do not use the bat file to modify plugins.ini .
Just activate your mods in MO then in in game click on mods than hit T on keyboard activate your mods one by one.
Than hit escape twice from main game menu click continue or select your prefered save it really is that easy. I have been playing for two days not staight but stop reload several times.
Good luck.

**Comment by AikoTM** [comment]
I noticed that you need to keep the plugins.txt open after the Batch is done and opens it.
Dunno why.. it worked before without it, but since yesterday it only works if i keep it open.

**Comment by Astor1977** [comment]
Still only partly works.  The Better Sorting mod only partly works, body is invisible, etc.  Haven't even played the 1.5 yet and i've gone from jubilant to really hating it.

**Comment by kranky1260** [comment]
I had the same problem with covenant walls mod brought it to the bottow of my load order and now my walls are visible again.

**Comment by AikoTM** [comment]
This honestly sounds like missing ArchiveInvalidation - although i doubt that someone who uses MO, doesn't know about this.

Did you edit the MO INI files or the Vanilla ones? I think the MO ones don't work yet. 
I just added the lines to all INI files.. even the custom one. Just in case..
May be worth checking if not already done.

**Comment by Astor1977** [comment]
Thanks, Aiko, will check into that.  I know the MO ini's are set, will double check the default.

**Comment by michaelrw10** [comment]
the reason that the batch file "doesn't work" is because MO will change the plugins file when you launch the game. Sometimes this happens. However, sometimes it does not revert the changes made by the batch file. So, if it doesnt work, try again. Sometimes I load the game only to see that not of the mods are enabled. I exit back to desktop, run the batch file again, then launch again.. usually it works on the second try. Rarely does it take 3 tries.

Also, I am going to look into making the file read only after it makes the changes.  If that works, I will try adding that command into the batch file so it is automatically done for you. Will keep you up to date

**Comment by Astor1977** [comment]
I tried right clicking and making it "read only", but it doesn't save it.  I think MO clears the read only flag.

**Comment by michaelrw10** [comment]
yeah, @Astor1977 , I took a closer look at the batch file and it already has code in it to enable the read-only file attribute.. but, like you said, MO reverts it. 

On a different note, I started looking at the course code to see how the plugins.txt file is generated.. If it was written in Python, I may have a shot at fixing it.. but that function is written in C++. I tried to make sense of it, but it would require too much for me to be able to get it going. That being said, the good news is, when @Tannin / @TanninOne sees this change, it will be a very simple implementation. It merely requires that an asterisk is appended to the start of each plugin name.. which, in terms of coding, is quite simple (for someone like Tannin). We just have to hang tight. I also imagine that other programs that read plugins.txt will also need to changed as well, namely Wrye and xEdit. In the meantime, they still work with MO fine because they are reading the old format, which MO still has (for now)

**Comment by AikoTM** [comment]
MO probably just creates a whole new file.

**Comment by pralima87** [comment]
So no fix for now.
Lets hope it doesnt take too long.
I just cant endure NMM

**Comment by KeltecRFB** [comment]
He may be either busy or waiting to see what Bethesda does next.  The batch file listed above works great.

**Comment by pralima87** [comment]
Can someone properly explain how the batch thing works?
Also is LOOT still working for you guys?
Mine sorts the plugins just fine but the M.O.2 list remains untouched.

**Comment by AikoTM** [comment]
Dumb Question..
You did start LOOT via MO2, right?

Isn't the Sort Function of MO a lite version of LOOT?

**Comment by pralima87** [comment]
I Added LOOT to the M.O.2 executable apps.
Just like FO4Edit.
I'm running it from there.
It sorts the list but after closint it M.O.2 list stays the same.
Like LOOT and M.O.2 are not communicating

**Comment by itunlimited** [comment]
@pralima87 use cranky 1260's steps above. Works great for, other than having to enable mods each time in FO4. 

I can't help with loot, I have too many mods to even think about using it.

**Comment by KeltecRFB** [comment]
I don't use the LOOT because I find LOOT not to be updated all the time.  It is better just to learn the order yourself or find a link to refer to such as [https://taleoftwowastelands.com/content/load-order-general-guide](Load Order Guide).  Of course, change the Game ESM according to FO4.

**Comment by pralima87** [comment]
Ok I"m really trying to follower "cranky 1260" guide but I can't enable the mods while ingame.
The "Mods" menu is not working.
It shows a server error :/....

**Comment by kstarler** [comment]
Regarding the LOOT question, there is a workaround posted, but the long and short is, make sure that MO is unlocked after opening LOOT. Also, you need to change the plugin list in MO to a different view tab (I always switch it to Data) before sorting in LOOT. Once you've finished using LOOT, when you switch back to the plugin list in MO, the .esm/.esp order should be changed to match LOOT. Note that changing the view after sorting in LOOT will not work. The view needs to be changed before sorting, or MO will overwrite the LOOT changes.

Alternatively, I have also had success creating a desktop shortcut for LOOT that runs through MO. Running LOOT through the MO shortcut without actually opening MO has worked this way to sort plugins, though it sometimes shows only Fallout.esm. When this happens, closing and re-opening LOOT via the MO shortcut resolves the issue.

**Comment by KeltecRFB** [comment]
FYI, 1.5.157 is out of beta and has been released.  Knuckleheads didn't remove the asterisk for enabled mods so the next MO beta release will still have to have the asterisks added to enable mods.

**Comment by pralima87** [comment]
Even a small hotfix for the plugins issue would be great already.
Right now enabling all the mods ingame works but is a pain...

**Comment by KeltecRFB** [comment]
He will when he gets to it.  Remember, we don't pay him.  And it is not really that much of a pain.

**Comment by pralima87** [comment]
Yep.
M.O is just awesome!

**Comment by manguz** [comment]
patience
He's here.
https://github.com/TanninOne?tab=activity

Thank you very much Tannin

**Comment by ogrotten** [comment]
So what's the suggested fix here? I've seen like 4 different ones in this thread.

**Comment by AikoTM** [comment]
I'm using the Batch File mentioned in @VijuaruKei 's post.
Changed it as suggested in the post from @michaelrw10 (Method 2).

**Comment by TanninOne** [comment]
MO 205 beta released which should fix this. Please test.

**Comment by manguz** [comment]
wow thank you very much

**Comment by pralima87** [comment]
Thanks a lot for the update!
But I'm getting error of missing LIBEAY32.DLL and SSLEAY32.DLL

**Comment by LawLeewer** [comment]
Same thing here /\
And thanks for the update, tannin

**Comment by pStyl3** [comment]
Thanks a lot Tannin. I can confirm the above mentioned issue, though for me, only "ssleay32.dll" is indicated as missing.

**Comment by TanninOne** [comment]
please try extracting this file to your MO folder
[openssl.zip](https://github.com/TanninOne/modorganizer/files/255758/openssl.zip)

**Comment by manguz** [comment]
used the zip in MO folder now only SSLEAY32.DLL is still missing

**Comment by LawLeewer** [comment]
Still getting the SSLEAY32.DLL missing.   I tried downloading SSLEAY32.DLL from an outside source and got a "0xc000007b" error.

**Comment by pStyl3** [comment]
No change for me, though I didn't get the missing libeay32.dll message anyway.

LawLeewer, better not try such things yourself, wait for Tannin fixing it properly.

**Comment by chazwarp923** [comment]
@LawLeewer rename the "libssl.dll" to "ssleay.ddl"

**Comment by LawLeewer** [comment]
Thanks, now it is working.

**Comment by pStyl3** [comment]
Renaming libssl.dll to ssleay32.dll worked for me.

**Comment by Astor1977** [comment]
TY TY TY TY TY TY TY TY!!!  :dancers:

**Comment by VijuaruKei** [comment]
Thank you for the update !

**Comment by AikoTM** [comment]
Installed the BETA into a new Folder and kept the Alpha installed.
Everything working fine for me so far. Didn't have to install or copy anything else...
MO2 BETA 1 found all my mods, added .exe (Bodyslide, F4SE) on its own.

Started the Game and all my mods were active.

Thank you very much Tannin.

**Comment by sbroadbent** [comment]
In followup to my last post, I disabled everything under the Plugins tab on the right side pane, so only Fallout4.esm, DLCRobot.esm, and DLCworkshop01.esm are loading.

Loading any of my saves with MO2.05 and Fallout 4 v1.5 results in the Crash to Desktop just after the loading screen.

Separately I installed F4SE so that I'd be able to use the Achievements mod, and I get the windows "Fallout 4 has stopped working".  Fault Module Name: "usvfs_x64.dll", similar to reported issue #399.  I'm on Windows 7, and I don't have any Antivirus software running.

After verifying the Integrity of the game cache on Steam, Steam indicates that all files validated, but I might just do a fresh reinstall instead.

I'm hoping that there might be a fix or some other resolution by the time Far Harbor on the 19th gets released.

**Comment by AikoTM** [comment]
Did you try to create a new game instead of loading an old save?
The CTD can be caused by missing Mods.

**Comment by TheGloriousArgonaut** [comment]
Hello , I am an avid user of MO from the TES series, and was exited to try out the 2.05 beta build of MO but to my dismay, I stumbled upon the recent issue MO is having with the games latest updates.

I have been reading this thread carefully, trying to get MO to work with the current fixes provided but I have not had much success.

I'm using the lastest download of FO4 and MO2.05 and am currently am trying the .bat file method to fix my game, I edited the line inside the .bat and successfully ran the bat. My plugins.txt now shows my esp list with 2 * in front of the listed mods.

I am launching the game through F4SE, but still do not see any mods working.

One comment I saw mentioned  going into the MODS tab and activating the mods. I have done this, and bringing up the LOAD ORDER is showing my mods checked. Trying to leave the mods menu is saying "mods are still downloading". Do I wait until that message no longer appears while trying to exit to menu? I see no progress bar for the "downloading mods".

**Comment by TheGloriousArgonaut** [comment]
Well I am giving up for now, if there is a more concise steps for fix please post it, there seems to be some sort of fix to get 2.05 and FO1.05 to work but it currently eludes me :(

**Comment by sbroadbent** [comment]
With MO2.05 and a fresh install of Fallout 4 1.5 (deleting the entire Fallout 4 folder and having Steam reinstall), Immediate Crash to Desktop when starting a new game (right after it asks to enable Survival Mode).  I did the same thing with all mods disabled.  Crash to Desktop.

Running Fallout 4 without Mod Organizer I can start a new game without any crashing, so there's something going on regarding Fallout 4 having an issue with MO2.05 for me.

**Comment by AikoTM** [comment]
@TheGloriousArgonaut - You no longer need th BAT file with MO 2.05.

@sbroadbent - Only difference between us 2 that i can spot is that i'm running Win 10 and you are running Win 7. As to how much this has any impact on your issues, i'm not sure. Make sure MO2 actually starts the fresh installed 1.05 FO4 and not the old 1.04 one you have on your system.

I don't want to tell you every obvious step you probably already have tried out.. i know how annoying that can be. Just going down the list of possible causes.

**Comment by TheGloriousArgonaut** [comment]
@AikoTM Could you post the link to the required .ini changes? Im guessing they are the same .ini changes that need to be made with NMM? Also is MO2 2.05 and FO4 1.05 working at all? Ill delete the BAT file in the mean time, and regenerate my plugins .txt.

So what is the solution? What is the current fix? Is there a working fix at all at this point?

**Comment by Astor1977** [comment]
@TheGloriousArgonaut I don't know about NMM (after MO, there is no other program), as for MO 2o5 and FO 1.5, i'm not having any discernible issues with them working together.  MO is pretty much just "plug & play" beyond the file renaming mentioned several posts above.  The only issues i'm having is LOOT still not really sorting things and Wrye not working.

**Comment by TheGloriousArgonaut** [comment]
@Astor1977 Which OS are you using? So you're saying just basic MO install is working without issues? You just launch F4SE and your mods are loading okay? You didnt do anything special to get it to work right?

**Comment by AikoTM** [comment]
Ini changes are the same. 
http://wiki.tesnexus.com/index.php/Fallout_4_Mod_Installation

MO 2.05 works fine with 1.05 for me. F4SE 1.5.157 aswell.
You are launching F4SE via MO2, right?

**Comment by TheGloriousArgonaut** [comment]
@AikoTM Yes currently launching F4SE via MO2. I installed the .zip and renamed the files like the instructions said and now I'm getting the error It was apprently trying to fix, its saying .ddl is missing. SO i am going to backup my plugins.txt and Loadorder and going to try a fresh install of 2.05.

Another question ALL of MO's files are in the same folder correct, currently I seemed to have half of MO;s files in  

C:\Program Files\ModOrganizer

and the other half in 

C:\Users\Administrator\AppData\Local\ModOrganizer\Fallout 4

this fresh install ill try and make sure all files are in the program files

**Comment by TheGloriousArgonaut** [comment]
Seems like fresh install worked, no more issues, really wierd.

**Comment by TheGloriousArgonaut** [comment]
I got the game to fire up with my mods one time. Then after noticing some potential Load order errors, I start get getting an immediate CTD on launching F4SE.

I disable all mods, same result.
I try launching JUST Fallout4.exe and the launcher same result.

I backup my mods again. Do another clean install of 2.05, reinstall F4SE , verify integrity of game cache for FO4 original files, seems like suddenly I cannot launch any instance of FO4 through MO. FIle paths for F4SE and fallout .exe's are corroect.

i get this error; 

> Problem signature:
>   Problem Event Name:	BEX64
>   Application Name:	Fallout4.exe
>   Application Version:	1.5.157.0
>   Application Timestamp:	571fa7c7
>   Fault Module Name:	StackHash_578f
>   Fault Module Version:	0.0.0.0
>   Fault Module Timestamp:	00000000
>   Exception Offset:	000007fefd674230
>   Exception Code:	c0000005
>   Exception Data:	0000000000000008
>   OS Version:	6.1.7601.2.1.0.256.1
>   Locale ID:	1033
>   Additional Information 1:	578f
>   Additional Information 2:	578fe98623d4ddb6b61de76532f618ea
>   Additional Information 3:	93e6
>   Additional Information 4:	93e6a07ecb6b89d166d238c955e1ec92
> 
> Read our privacy statement online:
>   http://go.microsoft.com/fwlink/?linkid=104288&clcid=0x0409
> 
> If the online privacy statement is not available, please read our privacy statement offline:
>   C:\Windows\system32\en-US\erofflps.txt

Ughhh I JUST had it working I tweaked the order of  3 mods then catastrophe. I WILL MAKE THIS WORK DAMMIT xD

**Comment by sbroadbent** [comment]
@AikoTM I'm pretty sure it's using the proper folder, especially since the main menu is different for each with the added options in 1.5.  I have two folders named "Fallout 4 1.4" and "Fallout 4 1.5", except whichever version is active is just named "Fallout 4".  When I switch versions, I add the version number to the "Fallout 4" folder, and then delete the version number for the version I want to be active.

I appreciate any suggestions for things to try to get this to work.

Has anyone had any issues with switching profiles in MO2.05?  I tried creating a new profile in MO2.05 to create a fresh profile to see if that might help, but when I switch to it, Mod Organizer stops responding.  When I load up MO2.04 it switches profiles without any issues.

**Comment by AikoTM** [comment]
@TheGloriousArgonaut 
"I installed the .zip and renamed the files"
I'm not sure how to understand this..

Which and how did you install the ZIP and what files did you rename?

**Comment by AikoTM** [comment]
@sbroadbent - I'm currently at a loss regarding your issues. Did MO2 create any log files?

**Comment by TheGloriousArgonaut** [comment]
Naw i figured it out again, it didn't like where I installed the MO root folder, I moved it off the /c: programs and its fine now. was just wierd kuz it fired up flawlessly once on the other drive but after that it was just like NOPE.

**Comment by sbroadbent** [comment]
I presume any logs MO2 creates would show up in the logs folder where MO2 is installed?  Currently there's nothing in that folder.  I decided to do an uninstall of MO2.05, and reinstall but that hasn't had any effect.

As much as I don't want to, I might have to temporarily install NMM to experience the wonders of the 1.5 patch and survival mode...

**Comment by Seireitou** [comment]
Hello, I don't know if this is an old problem but It seems I don't or even can't experience the problems you guys are having because I can't get anything to launch through MO. I tried looking around but I couldn't find any solution. So the game works perfectly without MO. No crashes, nothing. When launching with MO, it does NOTHING. F4SE just launches a CMD window that stays there no matter how long you wait. The same is with the launcher and actually fallout4.exe (No CMD window though) Could someone please walk me through this? I've been using MO for years and I really don't wanna move to NMM. Thank you for any help you can provide.

**Comment by TheGloriousArgonaut** [comment]
@AikoTM the zip and files to rename was an old fix provided by TanninOne in this thread,for ppl expereincing a problem that I assume now was to fix a 2.04 issue. I just tried it becuase I was confused about my own issue.

@Seireitou have you tried moving the root folder of MO out of /:c programs? I just put my MO in its own seperate program nect to the programs file, not in it, and MO started working fine. Also have you changed your custom.ini located in my documents? you need to add a line to that to get MO to work properly. Also do you have F4SE installed and linked properly in MO?

If its none of those 3 issues im stumped :/

**Comment by Astor1977** [comment]
" you need to add a line to that to get MO to work properly"  Uhh... What line?

**Comment by AikoTM** [comment]
He is speaking about the INI changes, i assume.
http://wiki.tesnexus.com/index.php/Fallout_4_Mod_Installation

**Comment by sbroadbent** [comment]
I have figured out my issue.  Turns out it was a mod, NoDotDotDot (http://www.nexusmods.com/fallout4/mods/425/?), which exists to remove the "..." from the end of long item names.

I initially went through the process of installing NMM (so painful...), and loading up my entire mod list in it, but the problem persisted.  Deleted everything, that was installed in NMM, and started installing mods one-by-one, until I came to NoDotDotDot.  Jumped back to MO2.05, disabled NoDotDotDot, and can load my existing save games in FO1.5.

I have a feeling the problem was that although in my MO2.05/FO1.5 tests I had disabled all my plugins (on the right side pane, not on the left side pane), because NoDotDot doesn't actually have a plugin, it was still being loaded... so in otherwords, user error.

Now my only issue is that whenever I try to switch profiles in MO2.05, it stops responding, but that is better off being reported in a new issue.
